src/libs/actions/IOU.js

```diff
@@ -156,14 +186,13 @@ function createIOUSplit(params) {
  * @param {String} params.reportID
  */
 function createIOUSplitGroup(params) {
-    Onyx.merge(ONYXKEYS.IOU, {loading: true, creatingIOUTransaction: true, error: false});
+    startLoadingAndResetError();
 
     API.CreateIOUSplit({
         ...params,
         splits: JSON.stringify(params.splits),
     })
-        .then(() => Onyx.merge(ONYXKEYS.IOU, {loading: false, creatingIOUTransaction: false}))
-        .catch(() => Onyx.merge(ONYXKEYS.IOU, {error: true}));
+        .then(() => Onyx.merge(ONYXKEYS.IOU, {loading: false, creatingIOUTransaction: false}));
```

Comment:
How is error here being handled?

Reply:
Which error do we need to handle? Where's it at?

Reply:
I guess these maybe? https://github.com/Expensify/Auth/blob/d3cf26e0330c3a4cf98c6ac3abf141a53f617f33/auth/command/CreateIOUSplit.cpp#L15-L18
Should hit the standard handler https://github.com/Expensify/Web-Expensify/blob/7ecf0d26981bcf715f551c58a21744750c06fc83/lib/Auth.php#L115

And because they are 404 and we removed the default handler that rejects the promise we must now do something?